toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_xnvme.py

```diff
@@ -0,0 +1,309 @@
+"""
+This is a port of the tests:
+
+xnvme_enum.sh         --> test_enum()
+xnvme_enum_fabrics.sh --> test_enum_fabrics()
+xnvme_feature_get.sh  --> test_feature_get()
+xnvme_feature_set.sh  --> test_feature_set()
+xnvme_format.sh       --> test_format()
+xnvme_sanitize.sh     --> test_sanitize() -- TODO: Needs some investigation
+xnvme_idfy.sh         --> test_idfy()
+xnvme_idfy_ctrlr.sh   --> test_idfy_ctrlr()
+xnvme_idfy_ns.sh      --> test_idfy_ns()
+xnvme_info.sh         --> test_info()
+xnvme_library_info.sh --> test_library_info()
+xnvme_log-erri.sh     --> test_log_erri()
+xnvme_log-health.sh   --> test_log_health()
+xnvme_log.sh          --> test_log()
+xnvme_padc.sh         --> test_padc()
+xnvme_pioc.sh         --> test_pioc()
+xnvme_sanitize.sh     --> MISSING
+
+Observation:
+    How was idfy-cs tested previously?
```

Comment:
I don't think this should be here

Reply:
I went with removing all of these comments and adding them to the commit-message instead.